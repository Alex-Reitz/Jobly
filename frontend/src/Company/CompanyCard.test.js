import React from "react";
import { render } from "@testing-library/react";
import Card from "./CompanyCard";
import { MemoryRouter } from "react-router";
import UserContext from "../Context/userContext";

it("matches snapshot", function () {
  const testFunc = jest.fn();
  const info = jest.fn(() => 123);
  const { asFragment } = render(
    <MemoryRouter>
      <UserContext.Provider value={{ currentUser: testFunc }}>
        <Card info={info} />
      </UserContext.Provider>
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
