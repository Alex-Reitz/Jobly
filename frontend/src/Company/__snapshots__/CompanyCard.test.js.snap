// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="content"
  >
    <p
      class="company-name-link"
    >
      <strong>
        Company Name: 
      </strong>
      <a
        href="/companies/undefined"
      >
         mockConstructor
      </a>
    </p>
    <p
      class="company-description-card"
    >
      <strong>
        Description: 
      </strong>
    </p>
  </div>
</DocumentFragment>
`;
